framework/include/utils/NestedSolve.h

```diff
@@ -303,7 +309,7 @@ NestedSolve::nonlinear(V & guess, T compute)
   CorrespondingJacobian<V> jacobian;
   sizeItems(guess, residual, jacobian);
 
-  std::size_t n_iterations = 0;
+  n_iterations = 0;
```

Comment:
```suggestion
  _n_iterations = 0;
```

Reply:
If I agree with this suggested change and click on "Commit suggestion", does it automatically change my code to the suggested one and no other action needs to be taken?